lib/screen/home/home_page.dart

```diff
@@ -426,6 +414,22 @@ class HomePageState extends State<HomePage>
     );
   }
 
+  Widget _carouselTipcard(BuildContext context) {
+    final configurationService = injector<ConfigurationService>();
+    return MultiValueListenableBuilder(
+      valueListenables: [
+        configurationService.showTvAppTip,
+        configurationService.showCreatePlaylistTip,
+        configurationService.showLinkOrImportTip,
+      ],
+      builder: (BuildContext context, List<dynamic> values, Widget? child) {
+        return CarouselWithIndicator(
```

Comment:
I think you can get the enable status from `values` and then pass it into the child function to check. So you won't need to call the configuration service again to get them.